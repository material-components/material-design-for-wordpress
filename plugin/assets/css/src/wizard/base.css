/**
 * Copyright 2020 Google LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

.material-wizard {
	--mdc-typography-headline5-font-family: "Roboto Mono", monospace;
	--material-design-wizard-primary-color: #212121;
	--material-design-wizard-primary-text: #fff;
	max-width: 1024px;

	& .mdc-button {
		--mdc-theme-primary: var(--material-design-wizard-primary-color);
		--mdc-theme-on-primary: var(--material-design-wizard-primary-text);
	}

	& h3,
	& h4 {
		margin-top: 0;
	}
}

.material_page_material-onboarding-wizard {

	& #adminmenumain,
	& #wpadminbar,
	& #wpfooter,
	& .update-nag {
		display: none;
	}

	& #wpcontent {
		margin-left: 0;
		padding-left: 0;
	}
}

#material-onboarding-wizard {
	--mdc-typography-font-family: "Roboto", sans-serif;
	--mdc-typography-headline3-font-family: "Roboto", sans-serif;
	--mdc-typography-headline4-font-family: "Roboto", sans-serif;
	--mdc-typography-body1-font-family: "Roboto", sans-serif;
	--mdc-typography-button-font-family: "Roboto", sans-serif;
	margin-top: -32px;
}
