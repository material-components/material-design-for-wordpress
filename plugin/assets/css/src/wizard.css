/**
 * Copyright 2020 Google LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

@import "@material/typography/dist/mdc.typography.css";
@import "@material/layout-grid/dist/mdc.layout-grid.css";
@import "@material/button/dist/mdc.button.css";
@import "@material/switch/dist/mdc.switch.css";
@import "@material/ripple/dist/mdc.ripple.css";
@import "@material/circular-progress/dist/mdc.circular-progress.css";
@import "./wizard/index.css";
