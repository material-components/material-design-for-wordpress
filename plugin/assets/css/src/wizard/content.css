/**
 * Copyright 2020 Google LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

.material-wizard__content {
	background-color: #fff;
	box-shadow: 0 2px 2px rgba(0, 0, 0, 0.15);
	padding: 2.5rem;

	& p {
		font-family: var(--mdc-typography-body1-font-family, var(--mdc-typography-font-family, Roboto,));
		font-size: var(--mdc-typography-body1-font-size, 1rem);
		line-height: var(--mdc-typography-body1-line-height, 1.5);
		font-weight: var(--mdc-typography-body1-font-weight, 400);
		letter-spacing: var(--mdc-typography-body1-letter-spacing, 0.03125em);
		text-decoration: var(--mdc-typography-body1-text-decoration, inherit);
		text-transform: var(--mdc-typography-body1-text-transform, inherit);
	}

	& hr {
		margin: 2.5rem auto;
		width: 80%;
	}

	& .material-wizard__title {
		margin-top: 0;
	}

	& .material-wizard__card img {
		width: 100%;
	}
}
