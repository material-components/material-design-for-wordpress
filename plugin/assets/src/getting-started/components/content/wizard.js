/**
 * Copyright 2020 Google LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

/**
 * WordPress dependencies
 */
import { __ } from '@wordpress/i18n';

/**
 * Internal dependencies
 */
import Button from '../../../wizard/components/navigation/button';
import getConfig from '../../get-config';

export const Wizard = props => {
	return (
		<>
			<h2 className="material-gsm__content-title mdc-typography--headline3">
				{ __(
					'Install the Material Design Theme and quick start examples',
					'material-design'
				) }
			</h2>

			<p>
				{ __(
					'The theme applies Material Design principles and Material Theming to your site, so you can customize built-in WordPress elements like your siteâ€™s header and footer. Example layouts include pages for home, about, projects, blog, and contact.',
					'material-design'
				) }
			</p>

			<img
				src={ `${ getConfig( 'assetsPath' ) }apply-your-theme.png` }
				alt=""
				style={ { maxWidth: '40%' } }
			/>

			<div style={ { height: '15px' } }></div>

			<div className="material-gsm__content-actions">
				<Button
					style="mdc-button--raised"
					text={ __( 'Re-run quick start', 'material-design' ) }
					trailingIcon="navigate_next"
					onClick={ props.handleClick }
					link={ getConfig( 'wizardUrl' ) }
				/>
			</div>

			<div style={ { height: '20px' } }></div>
		</>
	);
};
