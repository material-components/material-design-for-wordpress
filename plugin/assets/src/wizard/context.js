/**
 * Copyright 2020 Google LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

/**
 * WordPress dependencies
 */
import { createContext, useReducer } from '@wordpress/element';

/**
 * Internal dependencies
 */
import { reducer } from './reducer';
import { STEPS, STATUS, ADDONS } from './constants';

/**
 * Setups context to be used across the app
 */
const StepContext = createContext();

const { Provider } = StepContext;

/**
 * Default state of the world
 *
 */
const initialState = {
	addons: Object.keys( ADDONS ),
	active: STEPS.WELCOME,
	previous: [],
	status: STATUS.IDLE,
	error: {},
};

/**
 * Creates a wrapper in order to use our custom reducer
 *
 * @param {*} param Children to render
 */
export const StepProvider = ( { children } ) => {
	const [ state, dispatch ] = useReducer( reducer, initialState );

	return <Provider value={ { state, dispatch } }>{ children }</Provider>;
};

export default StepContext;
