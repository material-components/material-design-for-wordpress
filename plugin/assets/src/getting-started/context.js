/**
 * Copyright 2020 Google LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

/**
 * WordPress dependencies
 */
import { useReducer, createContext } from '@wordpress/element';

/**
 * Internal dependencies
 */
import { STATUS } from '../wizard/constants';
import { reducer } from './reducer';
import { TABS } from './constants';

/**
 * Setups context to be used across the app
 */
const TabContext = createContext();

const { Provider } = TabContext;

const tabs = Object.keys( TABS );

/**
 * Default state of the world
 *
 */
const initialState = {
	activeTab: 'CUSTOMIZE',
	completed: [ 'WIZARD' ],
	status: STATUS.IDLE,
	actionToInstall: null,
	error: {},
	themeStatus: 'install',
	contentStatus: 'install',
	tabs,
};

/**
 * Creates a wrapper in order to use our custom reducer
 *
 * @param {*} param Children to render
 */
export const TabProvider = ( { children } ) => {
	const [ state, dispatch ] = useReducer( reducer, initialState );

	return <Provider value={ { state, dispatch } }>{ children }</Provider>;
};

export default TabContext;
