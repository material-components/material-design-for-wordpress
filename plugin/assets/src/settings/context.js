/**
 * WordPress dependencies
 */
import { useReducer, createContext } from '@wordpress/element';

/**
 * Internal dependencies
 */
import { STATUS } from '../wizard/constants';
import { UPDATERS } from './constants';
import { reducer } from './reducer';
import getConfig from '../admin/get-config';

/**
 * Setups context to be used across the app
 */
const SettingsContext = createContext();

const { Provider } = SettingsContext;

const initialState = {
	status: STATUS.IDLE,
	apiStatus: getConfig( 'apiStatus' ),
	updates: {},
	availableUpdates: Object.keys( UPDATERS ),
	updaters: UPDATERS,
	error: {},
};

initialState.updates[ UPDATERS.FONTS.type ] = true;
initialState.updates[ UPDATERS.ICONS.type ] = false;

/**
 * Creates a wrapper in order to use our custom reducer
 *
 * @param {*} param Children to render
 */
export const SettingsProvider = ( { children } ) => {
	const [ state, dispatch ] = useReducer( reducer, initialState );

	return <Provider value={ { state, dispatch } }>{ children }</Provider>;
};

export default SettingsContext;
