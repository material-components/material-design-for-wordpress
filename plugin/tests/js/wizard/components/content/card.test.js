/**
 * Copyright 2020 Google LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

/**
 * External dependencies
 */
import '@testing-library/jest-dom/extend-expect';
import { render, cleanup } from '@testing-library/react';

/**
 * Internal dependencies
 */
import Card from '../../../../../assets/src/wizard/components/content/card';
import { StepProvider } from '../../../../../assets/src/wizard/context';

const setup = props => {
	return render(
		<StepProvider>
			<Card { ...props } />
		</StepProvider>
	);
};

describe( 'Wizard: Card', () => {
	afterEach( cleanup );

	it( 'matches snapshot', () => {
		const wrapper = setup( {
			image: 'http://example.com/image.png',
		} );
		expect( wrapper ).toMatchSnapshot();
	} );

	it( 'matches snapshot when switch is set', () => {
		const wrapper = setup( {
			image: 'http://example.com/image.png',
			switch: 'DEMO',
		} );
		expect( wrapper ).toMatchSnapshot();
	} );

	it( 'matches snapshot when imageSpan is set', () => {
		const wrapper = setup( {
			image: 'http://example.com/image.png',
			switch: 'DEMO',
			imageSpan: 6,
		} );
		expect( wrapper ).toMatchSnapshot();
	} );

	it( 'matches snapshot when contentSpan is set', () => {
		const wrapper = setup( {
			image: 'http://example.com/image.png',
			switch: 'DEMO',
			contentSpan: 6,
		} );
		expect( wrapper ).toMatchSnapshot();
	} );

	it( 'matches snapshot when children is set', () => {
		const wrapper = setup( {
			image: 'http://example.com/image.png',
			switch: 'DEMO',
			children: <div>Example content</div>,
		} );
		expect( wrapper ).toMatchSnapshot();
	} );
} );
