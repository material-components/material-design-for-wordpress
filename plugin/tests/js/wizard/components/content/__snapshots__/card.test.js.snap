// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wizard: Card matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4"
        >
          <img
            alt=""
            src="http://example.com/image.png"
          />
        </div>
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-8 mdc-layout-grid__cell--align-middle"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
    >
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4"
      >
        <img
          alt=""
          src="http://example.com/image.png"
        />
      </div>
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-8 mdc-layout-grid__cell--align-middle"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Wizard: Card matches snapshot when children is set 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4"
        >
          <img
            alt=""
            src="http://example.com/image.png"
          />
        </div>
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-7 mdc-layout-grid__cell--align-middle"
        >
          <div>
            Example content
          </div>
        </div>
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--align-middle"
        >
          <div
            class="mdc-switch material-wizard-switch material-wizard-switch__DEMO mdc-switch--checked"
          >
            <div
              class="mdc-switch__track"
            />
            <div
              class="mdc-switch__thumb-underlay"
            >
              <div
                class="mdc-switch__thumb"
              />
              <input
                aria-checked="true"
                class="mdc-switch__native-control"
                id="DEMO"
                role="switch"
                type="checkbox"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
    >
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4"
      >
        <img
          alt=""
          src="http://example.com/image.png"
        />
      </div>
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-7 mdc-layout-grid__cell--align-middle"
      >
        <div>
          Example content
        </div>
      </div>
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-switch material-wizard-switch material-wizard-switch__DEMO mdc-switch--checked"
        >
          <div
            class="mdc-switch__track"
          />
          <div
            class="mdc-switch__thumb-underlay"
          >
            <div
              class="mdc-switch__thumb"
            />
            <input
              aria-checked="true"
              class="mdc-switch__native-control"
              id="DEMO"
              role="switch"
              type="checkbox"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Wizard: Card matches snapshot when contentSpan is set 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4"
        >
          <img
            alt=""
            src="http://example.com/image.png"
          />
        </div>
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-5 mdc-layout-grid__cell--align-middle"
        />
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--align-middle"
        >
          <div
            class="mdc-switch material-wizard-switch material-wizard-switch__DEMO mdc-switch--checked"
          >
            <div
              class="mdc-switch__track"
            />
            <div
              class="mdc-switch__thumb-underlay"
            >
              <div
                class="mdc-switch__thumb"
              />
              <input
                aria-checked="true"
                class="mdc-switch__native-control"
                id="DEMO"
                role="switch"
                type="checkbox"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
    >
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4"
      >
        <img
          alt=""
          src="http://example.com/image.png"
        />
      </div>
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-5 mdc-layout-grid__cell--align-middle"
      />
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-switch material-wizard-switch material-wizard-switch__DEMO mdc-switch--checked"
        >
          <div
            class="mdc-switch__track"
          />
          <div
            class="mdc-switch__thumb-underlay"
          >
            <div
              class="mdc-switch__thumb"
            />
            <input
              aria-checked="true"
              class="mdc-switch__native-control"
              id="DEMO"
              role="switch"
              type="checkbox"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Wizard: Card matches snapshot when imageSpan is set 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6"
        >
          <img
            alt=""
            src="http://example.com/image.png"
          />
        </div>
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-5 mdc-layout-grid__cell--align-middle"
        />
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--align-middle"
        >
          <div
            class="mdc-switch material-wizard-switch material-wizard-switch__DEMO mdc-switch--checked"
          >
            <div
              class="mdc-switch__track"
            />
            <div
              class="mdc-switch__thumb-underlay"
            >
              <div
                class="mdc-switch__thumb"
              />
              <input
                aria-checked="true"
                class="mdc-switch__native-control"
                id="DEMO"
                role="switch"
                type="checkbox"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
    >
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-6"
      >
        <img
          alt=""
          src="http://example.com/image.png"
        />
      </div>
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-5 mdc-layout-grid__cell--align-middle"
      />
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-switch material-wizard-switch material-wizard-switch__DEMO mdc-switch--checked"
        >
          <div
            class="mdc-switch__track"
          />
          <div
            class="mdc-switch__thumb-underlay"
          >
            <div
              class="mdc-switch__thumb"
            />
            <input
              aria-checked="true"
              class="mdc-switch__native-control"
              id="DEMO"
              role="switch"
              type="checkbox"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Wizard: Card matches snapshot when switch is set 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4"
        >
          <img
            alt=""
            src="http://example.com/image.png"
          />
        </div>
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-7 mdc-layout-grid__cell--align-middle"
        />
        <div
          class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--align-middle"
        >
          <div
            class="mdc-switch material-wizard-switch material-wizard-switch__DEMO mdc-switch--checked"
          >
            <div
              class="mdc-switch__track"
            />
            <div
              class="mdc-switch__thumb-underlay"
            >
              <div
                class="mdc-switch__thumb"
              />
              <input
                aria-checked="true"
                class="mdc-switch__native-control"
                id="DEMO"
                role="switch"
                type="checkbox"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="material-wizard__card mdc-layout-grid__inner mdc-layout-grid__cell--align-middle"
    >
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-4"
      >
        <img
          alt=""
          src="http://example.com/image.png"
        />
      </div>
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-7 mdc-layout-grid__cell--align-middle"
      />
      <div
        class="mdc-layout-grid__cell mdc-layout-grid__cell--span-1 mdc-layout-grid__cell--align-middle"
      >
        <div
          class="mdc-switch material-wizard-switch material-wizard-switch__DEMO mdc-switch--checked"
        >
          <div
            class="mdc-switch__track"
          />
          <div
            class="mdc-switch__thumb-underlay"
          >
            <div
              class="mdc-switch__thumb"
            />
            <input
              aria-checked="true"
              class="mdc-switch__native-control"
              id="DEMO"
              role="switch"
              type="checkbox"
            />
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
